### DISCUSSION TITLE
admin return null when logged in

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#247] [Help Me!] [answered]
Ellie Walker

Have you searched for other help tickets? Yes

Describe your issue

I am trying to setup my admin login and private routes for the edit views. When I log in, admin is returning as null.

What have you tried?
console warning firebaseConfig.admin, process.env.REACT_APP_ADMIN_UID, and admin

Files in question:
https://github.com/elliewalkersm/personal-bio/blob/main/src/App/index.js

screenshots:
<img width="971" alt="Screen Shot 2021-05-24 at 8 01 36 PM" src="https://user-images.githubusercontent.com/76851756/119424738-a0f92e80-bccb-11eb-9b4b-a42779d363eb.png">

<img width="651" alt="Screen Shot 2021-05-24 at 8 01 57 PM" src="https://user-images.githubusercontent.com/76851756/119424725-9a6ab700-bccb-11eb-82c0-5609baffa4b7.png">

**Comment by drteresavasquez** [comment]
@elliewalkersm please share the resolution to your issue. Thanks!

**Comment by elliewalkersm** [comment, marked as answer]
I wasn't passing admin to my NavBar....ops!
